cupy/cuda/__init__.py

```diff
@@ -37,14 +36,6 @@
 except ImportError:
     thrust_enabled = False
 
-cub_enabled = False
-if int(os.getenv('CUB_DISABLED', 0)) == 0:
-    try:
-        from cupy.cuda import cub  # NOQA
-        cub_enabled = True
-    except ImportError:
-        pass
```

Comment:
👍 I thought this change would disable `from cupy.cuda import cub` but it seems still working. Interesting...